prowler/providers/aws/lib/security_hub/security_hub.py

```diff
@@ -284,3 +285,69 @@ def _send_findings_in_batches(
                 f"{error.__class__.__name__} -- [{error.__traceback__.tb_lineno}]:{error} in region {region}"
             )
             return success_count
+
+    @staticmethod
+    def test_connection(
+        session: Session,
+        region: str,
```

Comment:
I'm not sure about this, probably we will need to check all the regions where AWS Security Hub is enabled and return it. At least this is the way we are doing it now.